src/rely/QCheck_rely.rei

```diff
@@ -0,0 +1,24 @@
+/**
+  {1 Rely backend for QCheck}
+
+    We can also use environment variables for controlling QCheck here
+
+    - [QCHECK_SEED] if an integer, will fix the seed
+    - [QCHECK_LONG] is present, will trigger long tests
+
+    @since 0.9
```

Comment:
btw this should be `@since NEXT_RELEASE` (or 0.10, worst case)

Reply:
This brings up the question of what we want to do concerning release? Since Rely is not on opam, I imagine this isn't releasable there without needing to inject some module that would live in Rely. 

I could see releasing this to npm as qcheck-rely, or if you'd rather have it live elsewhere, we could move this to reason-native and release it as @reason-native/qcheck-rely

Reply:
I don't know how reason-native works, is it a kind of giant monorepo? Both options are ok for me, it's your code :slightly_smiling_face: 
If it's simpler to move it to reason-native it's totally fine by me.

Reply:
It's an npm namespace that we are using for a lot of the native reason development tools we are building at Facebook. It also happens to be a monorepo.

I'll think some more about where it makes sense to live.